exercises/practice/pov/test_cases.go

```diff
@@ -0,0 +1,156 @@
+package pov
```

Comment:
The (sub-)test names (and test cases) from the canonical test data https://github.com/exercism/problem-specifications/blob/main/exercises/pov/canonical-data.json should be respected. It's ok to add more tests but if it is very hard to match the code here to the canonical data we have a hard time updating the code when the canonical test data was improved.

Reply:
I assume you are referring to the "description" field and maybe also the "property" field of each of the "cases", is that correct?  I could  add this information as additional fields to the structs `fromPovTestCase` and `pathToTestCases`. Would that be sufficient?

Reply:
Pretty much. Since the people working on problem spec usually put quite some thought into making these names meaningful for the students, I would recommend you use the new description field in t.Run of `TestFromPov` and `TestPathTo` instead of your self-constructed names.